common/src/main/java/earth/terrarium/cadmus/common/claims/ClaimApiImpl.java

```diff
@@ -220,6 +224,9 @@ public boolean canEntityGrief(Level level, BlockPos pos, @NotNull Entity entity)
 
     @Override
     public boolean canPickupItem(Level level, BlockPos pos, ItemEntity item, @NotNull Entity picker) {
+        if (item.getItem().is(Cadmus.ALLOWS_CLAIM_PICKUP)) {
+            return true;
```

Comment:
Use a 1 line return statement for short checks like these. ex. `if(item.getItem().is(Cadmus.ALLOWS_CLAIM_PICKUP)) return;`
It makes the code cleaner imo especially when there's a lot of consecutive checks